client/components/domains/register-domain-step/index.jsx

```diff
@@ -360,6 +386,18 @@ class RegisterDomainStep extends React.Component {
 						{ this.renderSearchFilters() }
 					</CompactCard>
 				</StickyPanel>
+				{ showTldFilterBar && (
```

Comment:
Not too big of a deal, but since this block of code is repeated, we could return it from a function call and then call the function.

Reply:
Ah, good point - I hadn't considered that. Since we'll only be left with one block after the A/B testing is done, I think we're fine either way :)

Reply:
Yeah...totally. That's why I don't think it's a big deal. 👍